### ISSUE TITLE
Separate Build for Widescreen FOV & How To Compile

### ISSUE THREAD

**Issue by eXPc023** [mikkokko/csldr#44]
I'm requesting a separate build just for widescreen FOV correction support since many people have been looking for the ways of getting correct FOV for widescreen (Some people play the legacy build because HL25 doesn't work with their GPU for some reason).
If you install csldr, you will lose ability to run in software and D3D (although I you are working on it), and you will need to turn off a lot of settings that changes the bobbing of the weapon etc. Does widescreen FOV require explicit OpenGL support?
And the other request is to learn how to compile it from the source, because I couldn't compile it using VS 2015. Do I need a newer or older version?
Kind regards.

**Comment by mikkokko** [comment]
I'm not planning to make separate builds with different features because that would just complicate things. I've been meaning to set more sane default values for some cvars for a while now, just haven't gotten around doing it. fov_horplus doesn't need opengl at all, however it doesn't work properly in software mode because the viewmodel is not drawn when the fov is set to values over 90. Compiling should be as simple as `cmake -B build -A Win32` and `cmake --build build`. VS2013-VS2022 should work out of the box but I've been only building this with VS2022 lately.

**Comment by eXPc023** [comment]
Oh okay, I was able to generate the project but I was getting error while compiling it. I just needed to change it to "Release", well. But my reason for one version for widescreen FOV is: I'm playing the legacy steam_beta build of Half-Life and Counter-Strike because the new version "25" causes significant performance drop when smoke appears. It must be an issue with the engine. And since the legacy version doesn't have FOV adjustment, I have to set it manually or install csldr on supported games (CS 1.6/CZ). But there's no way to change FOV for games like:
Day Of Defeat
Deathmatch Classic
Condition Zero: Deleted Scenes
So it would be good to add support for every official Goldsrc game for people can't run the HL25 update. Maybe an ASI plugin or something? AFAIK they don't trigger VAC.
